pom.xml

```diff
@@ -26,13 +26,12 @@
                             <goal>npl-test</goal>
                             <goal>npl-api</goal>
                         </goals>
-                        <configuration>
-                            <enableCompilerOptimisations>true</enableCompilerOptimisations>
```

Comment:
Somewhat unrelated, but `npl-starter` is supposed to be lean so I see no reason to have this here.